coopertunes/supervisors/Audio2Mel.py

```diff
@@ -0,0 +1,20 @@
+from coopertunes.models import Audio2Mel
```

Comment:
wait, w want to train converting Audio2Mels?

Reply:
Yes, explanation above.